typescript/infra/scripts/utils.ts

```diff
@@ -196,6 +204,43 @@ export async function getMultiProviderForRole(
   return multiProvider;
 }
 
+export async function getKeysForRole(
+  txConfigs: ChainMap<ChainMetadata>,
+  environment: DeployEnvironment,
+  context: Contexts,
+  role: Role,
+  index?: number,
+): Promise<ChainMap<string>> {
```

Comment:
Any thoughts on returning `Promise<ChainMap<CloudAgentKey>>` instead? I don't feel strongly, it's just that we already have some existing abstractions for keys that we've used and and here we opt to just handle the privateKey directly, which is a slight departure

Reply:
Will do